app/javascript/components/upload/DifferentialExpressionFileForm.jsx

```diff
@@ -8,6 +8,10 @@ import { clusterFileFilter } from './ClusteringStep'
 import CreatableSelect from 'react-select/creatable'
 
 const allowedFileExts = FileTypeExtensions.plainText
+const requiredFields = [
+  { label: 'Associated annotation', propertyName: 'differential_expression_file_info.annotation_name'},
+  { label: 'Associated clustering file', propertyName: 'differential_expression_file_info.clustering_association'},
```

Comment:
To refine casing from what the screenshot shows is

> You must specify Associated annotation
> You must specify Associated clustering file

to

> You must specify associated annotation
> You must specify associated clustering file

```suggestion
  { label: 'associated annotation', propertyName: 'differential_expression_file_info.annotation_name'},
  { label: 'associated clustering file', propertyName: 'differential_expression_file_info.clustering_association'},
```

Reply:
I actually like the exact word usage so I can map it directly to a spot in the form, though in that case it might make sense to utilize semi-colons to make the sentence structure better like:

> You must specify: Associated annotation
> You must specify: Associated clustering file

Reply:
I like this better - I agree about matching the form case to the message, and this seems like a good middle ground that will benefit all forms.

Reply:
Excellent suggestion, Emily!  I agree.